config/app_configuration.go

```diff
@@ -9,12 +9,14 @@ import (
 )
 
 type AppConfiguration struct {
-	Name                string                   `yaml:"-" json:"name"`
-	ModuleSource        string                   `yaml:"module" json:"module"`
-	ModuleSourceVersion *string                  `yaml:"module_version,omitempty" json:"moduleVersion"`
-	Variables           map[string]any           `yaml:"vars" json:"vars"`
-	EnvVariables        map[string]string        `yaml:"environment" json:"envVars"`
-	Capabilities        CapabilityConfigurations `yaml:"capabilities" json:"capabilities"`
+	Name                string                 `yaml:"-" json:"name"`
+	ModuleSource        string                 `yaml:"module" json:"module"`
+	ModuleSourceVersion *string                `yaml:"module_version,omitempty" json:"moduleVersion"`
+	Variables           map[string]any         `yaml:"vars" json:"vars"`
+	Connections         core.ConnectionTargets `yaml:"connections" json:"connections"`
```

Comment:
One line 70, Connections should be passed into validation, right?
https://github.com/nullstone-io/iac/pull/4/files#diff-295058163fe8dac3bcc2360e6b0867465e79e90494b74819136250c7cb34fc6bL64

Reply:
I think you are right...
From the diffs it looks like this was missing before this PR. I have updated this to pass the connections are you pointed out. Good catch.